subcommittees/policy-CoC/README.md

```diff
@@ -0,0 +1,33 @@
+# The Code of Conduct Policy Subcommittee
+
+## Purpose and Scope
+
+Enforce the Code of Conduct (CoC) in all parts of the Carpentries community
+
+## What We Do
+
+* communicate with the Carpentries community about the CoC, as well as reporting and enforcement guidelines
+* review and respond to reports of CoC violations as deemed appropriate by the situation
+
+## Contact
+
+* policy@carpentries.org
+* confidential@carpentries.org
+
+## Current Members
+
+* Pauline Barmby - chair (community)
+* Chris Hamm (community)
+* Simon Waldman (community)
+* Erin Becker (staff)
+* Jonah Duckles (staff)
```

Comment:
@k8hertweck Maybe worth adding the the confidential@ email goes to C. MacDonnell (not a member of the committee). This independent reporting avenue has been used in the past and is a way for people to report issues to the Policy committee without needing to be identified to that group.